apps/docs/components/hooks/useScrollable.md

```diff
@@ -111,13 +175,17 @@ const { getContainerRef, state, getNextButtonProps, getPrevButtonProps } = useSc
 | --------- | --------------------- | ------------- | ----------- |
 <!-- vue -->
 | state  | `Ref<{hasPrev: boolean; hasNext: boolean;isDragged: boolean;}>` | `ref{ hasPrev: false, hasNext: false, isDragged: false }`  | Current state of scrollable  |
-| getContainerRef  | `Ref<>` |   | `Ref` to be attached on container  |
+| containerRef  | `Ref<>` |   | `Ref` to be attached on container  |
+| showNext  | `Function` |   | Method used to show next items in the container  |
+| showPrev  | `Function` |   | Method used to show previous items in the container  |
 | getPrevButtonProps  | `ComputedRef<{onClick, disabled}>` |   | ComputedRef to be binded onto previous button |
 | getNextButtonProps  | `ComputedRef<{onClick, disabled}>` |   | ComputedRef to be binded onto next button |
 <!-- end vue -->
 <!-- react -->
 | state  | `{hasPrev: boolean; hasNext: boolean;isDragged: boolean;}` | `{ hasPrev: false, hasNext: false, isDragged: false }`  | Current state of scrollable  |
-| getContainerProps  | `Function` |   | Props to be attched on container  |
+| containerRef  | `<>` |   | `Ref` to be attached on container  |
+| showNext  | `Function` |   | Method used to show next items in the container  |
+| showPrev  | `Function` |   | Method used to show previous items in the container  |
```

Comment:
Provide exact type signatures. `containerRef` is of type React.Ref<HTMLElement>, but you can say just `Ref` here. `showNext` and `showPrev` are both of type `() => void`. Same in Vue part.